xml/obs_admin_frontend.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="cha.obs.admin.frontend">
+ <title>Frontend</title>
+ <para>
+   The Ruby on Rails application is run through the apache web server with
+   <ulink url="https://www.phusionpassenger.com/">mod_passenger</ulink>.
+   You can control it vial <command>systemctl</command>
+<screen><command>systemctl {start, stop, restart} apache2</command></screen>
+ </para>
+ <section>
+  <title>Thinking Sphinx Search</title>
+  <para>
+    To be written...
+  </para>
+ </section>
+ <section>
+  <title>Delayed Jobs</title>
+  <para>
+    To be written...
+    obsapidelayed
```

Comment:
I would remove that line. In the docu it will look like
```
To be written... obsapidelayed
```
while all the other sections say
```
To be written...
```